packages/react-app-revamp/components/_pages/WizardFormCreateContest/Step3/useDeployContest.ts

```diff
@@ -84,6 +86,19 @@ export function useDeployContest(form: any) {
         chainId: chain?.id,
         hash: contract.deployTransaction.hash,
       });
+      if (
+        process.env.NEXT_PUBLIC_SUPABASE_URL !== "" &&
+        process.env.NEXT_PUBLIC_SUPABASE_URL &&
+        process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY !== "" &&
+        process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY
+      ) {
+        await indexContest({
```

Comment:
Oh question: can we not `await` here so that if the index fails it doesn't block deployment?